apps/frontend/services/api/kobo/useGetForms.ts

```diff
@@ -0,0 +1,52 @@
+import { DroughtDto, FloodDto, IncidentDto } from '@wfp-dmp/interfaces';
+import useSWR from 'swr';
+
+import { SearchFormData } from 'components/Filters/SearchFilters';
```

Comment:
a little weird to export a type from inside a component no?

Reply:
It is consistent with the other filters but agreed let's move then to a specific file in a another ticket